### PR TITLE
Add missing cache keys for test job.

### LINKED ISSUE #30: Missed Cache Key in PR #29
# 🐛 Bug Report

When #29 was merged the cache key for the `test` job was not updated to include `requirements.txt` like the `lint` job.

`lint` job:
https://github.com/cisagov/skeleton-docker/blob/b319fafc4777ecdcdba3ef2bfc9c50e777832223/.github/workflows/build.yml#L33-L36

`test` job:
https://github.com/cisagov/skeleton-docker/blob/b319fafc4777ecdcdba3ef2bfc9c50e777832223/.github/workflows/build.yml#L87-L88

### PR DESCRIPTION
**Pull request by felddy**
Fixes #30

# 

## 🗣 Description

Add missing cache keys for test job.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
💯

**Review by jsf9k — APPROVED** [review]
:zap:

**Comment by felddy** [comment]
@mcdonnnj does this look good to you?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -86,7 +86,9 @@ jobs:
         with:
           path: ${{ env.PIP_CACHE_DIR }}
           key: "test-${{ runner.os }}-\
-            ${{ hashFiles('**/requirements-test.txt') }}"
+            ${{ hashFiles('**/requirements-test.txt') }}-\
+            ${{ hashFiles('**/requirements.txt') }}-\
+            ${{ hashFiles('**/.pre-commit-config.yaml') }}"
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]:
```suggestion
            ${{ hashFiles('**/requirements-test.txt') }}-\
            ${{ hashFiles('**/requirements.txt') }}"
```
`.pre-commit-config.yaml` is only used by the `lint` workflow.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Thanks for getting this done. :godmode: Please see my one change.

**Review by mcdonnnj — APPROVED** [review]
💪💪💪

**Pull request merged**